README.md

```diff
@@ -0,0 +1,165 @@
+<div align="center">
+  <h1>Budget App</h1>
+</div>
+
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Budget-App ](#-budget-app-rails-)
+  - [🛠 Built With ](#-built-with-)
+    - [Key Features ](#key-features-)
+  - [🛠Live Preview ](#-live-preview-)
+  - [🛠 Description ](#-video-description-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Budget App <a name="about-project"></a>
+
+> This is a budget app that helps to keep track the expenses of your transactions and it builds using ruby on rails language it shows you every purchases you made and the amount of money you spent on each transaction and it also shows you the total amount of money you spent on all transactions.
+
+## 🛠 Built With <a name="built-with"></a>
+
+<h4> Tech Stack </h4>
+- Ruby on Rails
+- PSQL
+
+## 🛠 live Preview <a name="live-preview"></a>
+<!-- project live preview -->
+
```

Comment:
[ ] The live version of your app has some issues, kindly fix that 🤓
![error](https://github.com/saluumaa/Budget-App/assets/62156607/3db91c31-0944-4678-9a65-0e40b5319faa)